lib/metrics/context.js

```diff
@@ -156,6 +159,18 @@ module.exports = function (log, config) {
   }
 
   /**
+   * Attempt to clear previously-stashed metrics context metadata.
+   *
+   * @name clearMetricsContext
+   * @this request
+   */
+  function clear () {
+    return P.resolve()
+      .then(() => getMemcached().delAsync(getKey(getToken(this))))
+      .catch(() => {})
```

Comment:
Will this also silently gobble errors thrown from memcached?

So that your intention is obvious in the code, perhaps the catch should check the error, re-throwing if it's not an error we expected.